Dockerfile

```
@@ -3,13 +3,14 @@
 # Import docker image with maven installed
-FROM maven:3.8.3-openjdk-17 as builder 
-
+# FROM maven:3.8.3-openjdk-17 as builder 
+# Import docker image with Maven and JDK 17 (from Eclipse Temurin)
+FROM maven:3.8.3-eclipse-temurin-17 as builder
 # Set working directory
 WORKDIR /app
 
 # Copy source code from local to container
-COPY . /app
+COPY . /appFROM openjdk:17-jdk-alpine as deployer
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)